src/code/bank0.asm

```diff
@@ -340,16 +340,17 @@ ReplaceObjects56and57_trampoline::
 ; Specific data-copying routines
 ;
 
-; Copy $100 bytes without DMA (used on DMG), then switch back to bank at h
+; Copy $100 bytes without HDMA (used on DMG), then switch back to bank at h.
```

Comment:
It's actually using GDMA (yes, the registers are confusingly named) but HDMA is copying during HBlank, while GDMA just copies. Difference is bit 7 in HDMA5

Reply:
Hey, good catch, thanks. I updated the comments to reflect this.